src/types/plugin-input.ts

```diff
@@ -9,8 +9,12 @@ import { StaticDecode, Type as T } from "@sinclair/typebox";
  */
 export const pluginSettingsSchema = T.Object(
   {
-    configurableResponse: T.String({ default: "Hello, world!" }),
-    customStringsUrl: T.Optional(T.String()),
+    /**
+     * `owner/repo`: `response`
+     * `repo`: `response`
+     * `owner`: `response`
+     */
+    automatedResponses: T.Record(T.String(), T.String()),
```

Comment:
You should have some default like hello world

Reply:
I don't think it makes sense to have a default but I have used the devpool as that's where this is being used

Reply:
> I don't think it makes sense to have a default 

You need to adhere to the spec to get the reward for completing the task

> but I have used the devpool as that's where this is being used

Don't understand what this means